### ISSUE TITLE
How do I use regex to filter, but also filter out disabled branches?

### ISSUE THREAD

**Issue by smiles3983** [jan-molak/jenkins-build-monitor-plugin#427]
I have a regex pattern to filter the builds on the display, but I still see disabled branches.  I see that I can filter those out when I dont use a regex, but select the branches in the tree.  How can I use regex but also filter out disabled?

**Comment by dcendents** [comment]
it's easy to do with the View Job Filters plugin: https://wiki.jenkins.io/display/JENKINS/View+Job+Filters

**Comment by smiles3983** [comment]
> it's easy to do with the View Job Filters plugin: https://wiki.jenkins.io/display/JENKINS/View+Job+Filters

yeah, but will this filter out what I see in the Build Monitor Plugin?

**Comment by dcendents** [comment]
Yes it will

I use it to filter exactly what I want.

Here is my use case, it might show you how far you can push it:

- Add All jobs that were built in the last 4 hours
- Exclude the jobs if all of these builds were triggered by a timer
- Then keep only the develop branch
- Then add all feature branches (excluding main branches, develop and master)
- Then exclude all tags
- Then add all Failed, Unstable and Aborted jobs
- Then include all the jobs currently building or in the build queue
- Then exclude all disabled jobs

So I want the build monitor to show only our feature branches (while we work on a user story) and not show the develop or master branch. However they will be shown if the build failed or is unstable. The develop branch will also be shown for 4 hours after a feature branch is merged into it. develop is scheduled to build monthly, but these builds will not show up on the build monitor view (only while it builds, but will disappear straight away after if it is successful).

So in a jenkins where we have over 100 git projects, the build monitor will typically only show maixum 15-20 projects, which is what the developers are currently working on.
